src/unicode/utf8/utf8.gobra

```diff
@@ -0,0 +1,67 @@
+package utf8
+//+gobra
+
+import (
+	. "verification/utils/definitions"
+	sl "verification/utils/slices"
+)
+
+// utf8.Codepoints(s) returns the sequence of (utf-8 encoded) codepoints in `s`
+ghost
+pure
+ensures 0 < len(s) ==> 0 < len(res)
+ensures 0 == len(s) ==> 0 == len(res)
+trusted
+decreases
+func Codepoints(s []byte) (res seq[rune])
```

Comment:
is this function expected to be total? I would guess that we cannot convert any arbitrary sequence of bytes to a valid sequence of runes

Reply:
`Codepoints` behaves like [`utf8.DecodeRune`](https://pkg.go.dev/unicode/utf8#DecodeRune), which returns `RuneError` (`\uFFFD`) in case of invalid utf8. Therefore, `Codepoints` will have a `RuneError` rune in places where `s` had invalid unicode. Should I express this in the signature or will a comment suffice?

Reply:
To confirm what we discussed in our meeting, please add a comment